package.json

```diff
@@ -8,7 +8,9 @@
     "@testing-library/user-event": "^12.1.10",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-redux": "^7.2.5",
     "react-scripts": "4.0.3",
```

Comment:
- [ ] It would be better if we add `prop-types` to the package.json file. This will enable better coordination between teams working on a different parts of the same project